### PR TITLE
Implement app verification

### PR DESCRIPTION
**Pull request by razzeee**

### CODE REVIEW

File: frontend/src/asyncs/app.ts [hunk 1]
```diff
@@ -36,6 +41,22 @@ export async function generateUpdateToken() {
   }
 }
 
+/**
+ * Generates an update token for the logged in user.
```
Comment (jameswestman) [inline frontend/src/asyncs/app.ts:45]: Forgot to update the comment?

File: frontend/src/components/application/AppVerificationSetup.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                {({ open }) => (
+                  <div>
+                    {methodType.login_name} {methodType.login_provider}
+                  </div>
+                )}
+              </Disclosure>
+            )
+          }
+        })}
+        <Button onClick={() => verifyApp(app.id)}>{t("verify-app")}</Button>
```
Comment (jameswestman) [inline frontend/src/components/application/AppVerificationSetup.tsx:76]: The `verify` endpoint is only for "login_provider" verification. Website verification is automatic.

**Review by jameswestman — COMMENTED** [review]
We probably want to indicate if the app is currently verified, and if so, how. That's available through the `/verification/{appId}/status` endpoint.

Also, this might be better for a follow-up PR, but it would be nice to add instructions for how to verify an app with each method.

File: frontend/src/asyncs/app.ts (hunk 1 above)
Comment (razzeee) [reply frontend/src/asyncs/app.ts:45]: Yes :)

File: frontend/src/components/application/AppVerificationSetup.tsx (hunk 2 above)
Comment (razzeee) [reply frontend/src/components/application/AppVerificationSetup.tsx:76]: Moved inside that code path

**Comment by razzeee** [comment]
@jameswestman should be much better now, but I'm struggling to test this, as I only ever seem to see website verification and that stops working after half an hour or so.

I will probably need to setup a `story` for storybook, to finish this completly

**Comment by razzeee** [comment]
This should be ready now. Texts might need some fine tuning, but nothing, that can't be done in main.

**Pull request merged**